tests/testthat/test_examples.R

```diff
@@ -0,0 +1,52 @@
+context("test point alternative point ESS design (case 01)")
+
+test_that("plausible n2() function", {
+
+    order <- 9L
```

Comment:
if you want, you could scale the test down to 7L - maybe it converges faster then